.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: php
 
-sudo: false
```

Comment:
Ok, I have been confused this for a long time.
Would you please provide some reference about this?
Thanks!